charts/policy-controller/README.md

```diff
@@ -20,7 +20,9 @@ The following table lists the configurable parameters of the policy-controller c
 | policywebhook.extraArgs | object | `{}` |  |
 | policywebhook.image.pullPolicy | string | `"IfNotPresent"` |  |
 | policywebhook.image.repository | string | `"gcr.io/projectsigstore/policy-webhook"` |  |
-| policywebhook.image.version | string | `"sha256:5ddc8c4c7f609a6a8396f469fde685e8774f7ee3aa4ef56f18adcb90b546a48c"` |  |
+| policywebhook.image.version | string | 
+`"sha256:5ddc8c4c7f609a6a8396f469fde685e8774f7ee3aa4ef56f18adcb90b546a48c"` |  |
```

Comment:
should be all on one line.

Reply:
it doesn't seem to be a problem for the rendering.